### ISSUE TITLE
why is there not code tab for grid-12?

### ISSUE THREAD

**Issue by dominikwilkowski** [govau/designsystem#110]
I want to know the classes I can use?
A11y tab missing also, does this mean grid is not accessible?

**Comment by alex-page** [comment]
We removed the code and accessibility tab from certain components as they do not add value to the user in certain scenarios.

The grid component doesn't have accessibility or code tabs because:
- You can not keyboard navigate a grid
- You can not hear a grid with screen reader
- You can not see a grid with different color blindnesses
- We do not supply a react component for the grid. 
- We hoped the code examples below the variations would suffice.

If you disagree and have any good examples of how to display these I'm more then happy to think about implementing this.

**Comment by dominikwilkowski** [comment]
Where would I learn about the classes? Where would I learn about the sass mixins and functions each package contains? How will I be ensured about that particular components accessibility?

Is the `code` tab only for react?

**Comment by alex-page** [comment]
In the future if the grid had functions and mixins I would be more then happy to add it into a code tab. See here for an example of this:
https://designsystem.gov.au/components/core/code/

**Comment by dominikwilkowski** [comment]
But the grid does have global mixins and functions that it exposes: https://github.com/govau/uikit/blob/master/packages/grid-12/src/sass/_globals.scss

Also What you said:
> The grid component doesn't have accessibility or code tabs because:
> -   You can not keyboard navigate a grid
> -   You can not hear a grid with screen reader
> -   You can not see a grid with different color blindnesses
> -   We do not supply a react component for the grid.
> -   We hoped the code examples below the variations would suffice.

is a really good example of what should go into the a11y tab if you ask me :)

**Comment by alex-page** [comment]
No problem, happy to add those functions to the code page!

Still not entirely sure how we do the accessibility page when there is nothing to be shown, however I understand your point and i'll see what we can do.

**Comment by dominikwilkowski** [comment]
But I still think components that expose an API be it js or css should have a code tab which is all of them. I'd like to see an overview of all classes in a table much like your documentation for react. That way I can look things up quickly like a cheat sheet and don't have to run through each example.

And in terms of a11y tab: I think it's important to say: hey there is nothing that will impact a11y in this package rather than keep people guessing. Explicit over implicit :)

**Comment by alex-page** [comment]
There is now a code tab for the grid component. Thanks again @dominikwilkowski 😀 
https://designsystem.gov.au/components/grid-12/code/
